workflow_creation/advanced_branching_looping.md

```diff
@@ -1,2 +1,109 @@
-# Advanded workflow creation: Branching and looping
-to be written
\ No newline at end of file
+# Advanced workflow creation: Branching and looping
+
+To create more sophisticated workflows, the workflow editor offers branching and loops:
+
+Need to perform a case distinction and execute different services based on the result? Make use of the [branching feature](#branching).
+
+Want to execute a service multiple times? There is no need to manually call it multiple times, make use of the [loops](#loops).
+
+
```

Comment:
Suggestion: Add short note that these advanced topics have to be implemented using the xml notation, and say how to get that notation visible.